### PR TITLE
replace local adjustment script with external [TPU]

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

let's keep these updates at one place: https://github.com/Lightning-AI/utilities/pull/132

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @carmocca @borda @tchaton @justusschock @awaelchli

### CODE REVIEW

File: requirements/pytorch/adjust-versions.py
Comment (carmocca) [inline requirements/pytorch/adjust-versions.py:1]: This torch-specific piece of code should stay here, where we support pytorch. Not in utilities

Other repos can wget from this URL

File: dockers/base-ipu/Dockerfile
Comment (carmocca) [inline dockers/base-ipu/Dockerfile:1]: This file can be removed entirely

Comment (Borda) [reply dockers/base-ipu/Dockerfile:1]: addressing in https://github.com/Lightning-AI/lightning/pull/17583

File: requirements/pytorch/adjust-versions.py
Comment (Borda) [reply requirements/pytorch/adjust-versions.py:1]: btw, how many projects/repos we have that do not use PyTorch? :thinking:

Comment (carmocca) [reply requirements/pytorch/adjust-versions.py:1]: fabric, trainer or tm are fine. I would suggest here because it has the history

Comment (carmocca) [reply requirements/pytorch/adjust-versions.py:1]: 1, utilities

**Review by justusschock — APPROVED** [review]

File: requirements/pytorch/adjust-versions.py
Comment (Borda) [reply requirements/pytorch/adjust-versions.py:1]: That I very much agree; no PT dependency not even in scripts...

**Review by ethanwharris — APPROVED** [review]

**Review by lantiga — APPROVED** [review]

File: tests/tests_fabric/run_tpu_tests.sh [hunk 1]
```diff
@@ -13,8 +13,10 @@ fi
 
 echo "--- Install packages ---"
 # set particular PyTorch version
+pip install -q wget packaging
```
Comment (Borda) [inline tests/tests_fabric/run_tpu_tests.sh:16]: this skip is tricky, as it does not have parity with group check

**Pull request merged**